### DISCUSSION TITLE
Compare closing style

### DISCUSSION THREAD

**Discussion by SebastianStehle** [egil/AngleSharp.Diffing#30] [General]
Hi,

I have a bug report here: https://github.com/SebastianStehle/mjml-net/issues/161

The bug has not been detected by our tests because and seems to be related to self closing elements, so 

```
<v:image ...>
```

vs 

```
<v:image />
```

Is is possible to configure the comparer for that?

Furthermore: Do you have some ideas for these things?

https://github.com/SebastianStehle/mjml-net/blob/main/Tests/Internal/AssertHelpers.cs#L107

It is about conditional comments and how html entities are handled. Perhaps another option to the text comparer to normalize all entities before comparing them would work? I really dislike this part of my code.

**Comment by egil** [comment]
Was about to write that you could probably do this with a custom compare, but you beat me to it.

**Reply by SebastianStehle** [reply]
Forget the thing with the entity. I do not remember why it was actually there, because it looks like a bug from my side.
